deploy/lib/server_config.rb

```diff
@@ -989,9 +989,11 @@ def mlcp
       end
     end
 
+    @ml_username = @properties['ml.mlcp-user']
+    @ml_password = @properties['ml.mlcp-password']
```

Comment:
Just as an extra precaution, perhaps these should be:

@ml_username = @properties['ml.mlcp-user'] || @properties['ml.user']
@ml_password = @properties['ml.mlcp-password'] || @ml_password

Not everyone knows about or uses the ./ml upgrade feature and thus might not get those properties added to default.properties.